cmd/vulcan-blast-radius/main_test.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2022 Adevinta
+*/
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	checkstate "github.com/adevinta/vulcan-check-sdk/state"
+	"github.com/adevinta/vulcan-checks/cmd/vulcan-blast-radius/securitygraph"
+	report "github.com/adevinta/vulcan-report"
+)
+
+type intelAPIMock struct {
+	blastRadius func(req securitygraph.BlastRadiusRequest) (securitygraph.BlastRadiusResponse, error)
+}
+
+func (i *intelAPIMock) BlastRadius(req securitygraph.BlastRadiusRequest) (securitygraph.BlastRadiusResponse, error) {
+	return i.blastRadius(req)
+}
+
+func TestRun(t *testing.T) {
+	type args struct {
+		ctx            context.Context
```

Comment:
If `ctx` is always `context.Background()` I'd remove from the struct.

Reply:
yes makes sense